testsuite/omsimulator/Makefile

```diff
@@ -7,7 +7,6 @@ initialization.mos \
 initialization2.mos \
 outputState.mos \
 reset.mos \
-DualMassOscillator.mos \
```

Comment:
The file `DualMassOscillator.mos` is still there. Is the test not valid any more?

Reply:
@arun3688 `DualMassOscillator.mos` should have been moved down to `FAILINGTESTFILES` instead of deleting it from the Makefile. Please fix it.

The test is not running reliable because the gcc version doesn't flush the streams as it should.